src/chapters/configuring.xml

```diff
@@ -3009,13 +3009,102 @@ ed272546-87bd-4db9-acba-e36e1a9ca20a
                                             <primary>SSH</primary>
                                             <secondary>color scheme</secondary>
                                         </indexterm>The color scheme to use for the terminal
-                                        emulator used by SSH connections. Each color scheme dictates
-                                        the default foreground and background color for the
-                                        terminal. Programs which specify colors when printing text
-                                        will override these defaults.</para>
-                                    <para>This parameter is optional. By default, Guacamole will
-                                        render text as gray over a black background.</para>
-                                    <para>Possible values are:</para>
+                                        emulator used by SSH connections. It consists of a
+                                        semicolon-separated series of name-value pairs, separated by
```

Comment:
I'd suggest a slight tweak to this language - perhaps something like:

> It consists of a semicolon-separated series of name-value pairs, where each name-value pair is separate by a colon, ...

or something else that clarifies this a little bit.  My initial take on this is, "Well, is it separate by a semicolon or a colon?"  I get it when I look at the examples, but I think it has the potential to cause some confusion.

Reply:
Fixed